### PR TITLE
Added support for two clusters domain

### PR DESCRIPTION
**Pull request by marinakog**
Modified domain to create 2 clusters and check that WLS exporter produce metric which can be processed via Prometheus for both clusters.
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4111/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4137

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java [hunk 1]
```diff
@@ -715,46 +724,13 @@ public void tearDownAll() {
           .withFailMessage("uninstallNginx() did not return true")
           .isTrue();
     }
-
-    // shutdown domain1
-    logger.info("Shutting down domain1");
-    assertTrue(shutdownDomain(domain1Uid, domain1Namespace),
-            String.format("shutdown domain %s in namespace %s failed", domain1Uid, domain1Namespace));
-
     // delete mii domain images created for parameterized test
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java:727]: Should we check if SKIP_CLEANUP is set to true, we should avoid the clean up for a debug session,

**Review by anpanigr — COMMENTED** [review]
Looks good. Update the PR with Jenkin Result

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java (hunk 1 above)
Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java:727]: this should be done in cleanutil method, I just removed unnecessary delete in here

Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java:727]: this step need to be done before 

> Looks good. Update the PR with Jenkin Result

results are in description, I have updated with latest today

**Review by anpanigr — APPROVED** [review]
LGTM

**Review by bhavaniravichandran — APPROVED** [review]
LGTM

**Pull request merged**